tardis/apps/publication_workflow/static/publication-tokens/publication_tokens.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Tokens controller
+ *
+ */
+/*eslint no-unused-vars: ["error", { "argsIgnorePattern": "^_" }]*/
+angular
+.module('MyTardis')
+.controller('PublicationTokensController', function (ngDialog, $scope, $log, $http, $resource) {
+    var vm = this;
+    vm.tokensListRes = $resource('/apps/publication-workflow/tokens_json/:publication_id/', {}, {
+        'get': {method: 'GET', isArray: true}
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933951)
```suggestion
        "get": {method: 'GET', isArray: true}
```